bench/src/cli.rs

```diff
@@ -1,26 +1,43 @@
+use crate::{helpers::USER_KEYPAIR_PATH, strategies::Strategies, tx_size::TxSize};
 use clap::{command, Parser};
 
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 pub struct Args {
-    /// Number of tx(s) sent in each run
-    #[arg(short = 'n', long, default_value_t = 5_000)]
-    pub tx_count: usize,
-    /// Number of bench runs
-    #[arg(short = 'r', long, default_value_t = 1)]
-    pub runs: usize,
-    /// Interval between each bench run (ms)
-    #[arg(short = 'i', long, default_value_t = 1000)]
-    pub run_interval_ms: u64,
     /// Metrics output file name
     #[arg(short = 'm', long, default_value_t = String::from("metrics.csv"))]
     pub metrics_file_name: String,
```

Comment:
getting this strange error. when I do not provide that argument:

> thread 'main' panicked at bench/src/main.rs:30:48:
called `Result::unwrap()` on an `Err` value: error decoding response body: EOF while parsing a value at line 1 column 0

Caused by:
    0: error decoding response body: EOF while parsing a value at line 1 column 0
    1: EOF while parsing a value at line 1 column 0
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace